backend/services/store/menus/create.go

```
@@ -0,0 +1,93 @@
+	if err != nil {
+		return menus.MenuDisplay{}, err
+	}
+
+	err = s.imageRepository.PutObject(c, processedImage, entities.StoreImageObjectKey(imageObjectKey), contentType)
+	if err != nil {
+		return menus.MenuDisplay{}, services.ErrFailedToStoreImage
+	}
+
+	imageURL, err := s.imageURLGenerator.GenerateMenuImageURL(c, imageObjectKey)
+	if err != nil {
+		s.imageRepository.DeleteObject(c, entities.StoreImageObjectKey(imageObjectKey))
```

golangci-lint (2.12.2)

[error] Error return value of `s.imageRepository.DeleteObject` is not checked

(errcheck)